ui/main/src/app/modules/share/archives-logging-filters/archives-logging-filters.component.html

```diff
@@ -75,29 +75,29 @@
     <div class="opfab-vertical-bar"></div>
     <div class="opfab-publish-date" [formGroup]="this.parentForm">
         <div>
-            <of-datetime-filter filterPath="publishDateFrom" formControlName="publishDateFrom" [defaultDate]="defaultMinPublishDate" [defaultTime]="{ hour: 0, minute: 0}"
+            <of-datetime-filter id="opfab-publishDateFrom" filterPath="publishDateFrom" formControlName="publishDateFrom" [defaultDate]="defaultMinPublishDate" [defaultTime]="{ hour: 0, minute: 0}"
             labelKey="shared.filters." [maxDate]="publishMaxDate" (change)="onDateTimeChange($event)"></of-datetime-filter>
         </div>
         <div>
-            <of-datetime-filter filterPath="publishDateTo" formControlName="publishDateTo"
+            <of-datetime-filter id="opfab-publishDateTo" filterPath="publishDateTo" formControlName="publishDateTo"
```

Comment:
our ids are with dashes between words, not uppercase letters

Reply:
I put uppercase characters in this one (and the ones below) because there already was `id="opfab-processGroup"` and this wass the only field with a composed name in this file